doxia-modules/doxia-module-fo/src/main/java/org/apache/maven/doxia/module/fo/FoAggregateSink.java

```diff
@@ -1322,12 +1322,12 @@ private SinkEventAttributeSet getGraphicsAttributes( String logo )
         }
         catch ( IOException e )
         {
-            getLog().debug( e );
+            getLog().debug( e.getMessage() );
```

Comment:
Leave it as `e`, SFL4J and backends will do the rest for us.

Reply:
there is no constructor with Exception, i will change to e.getMessage(), e

Reply:
@slachiewicz @michael-o Proper code here would be ```getLog().debug("", e)```.  That assumes you want the full stack trace and not just the message.